scripts/genproto.sh

```diff
@@ -18,7 +18,7 @@ errcho() {
 
 
 if [ ! -f $protoc ] ; then
-    errcho "Could not find protoc in $protoc. Trying to use protoc in PATH."
+    echo "Could not find protoc in $protoc. Trying to use protoc in PATH."
     protoc=protoc # try loading from PATH
     if !(hash protoc 2>/dev/null) ; then
```

Comment:
unrelated, but while you're in this area, you [should](https://github.com/koalaman/shellcheck/wiki/SC1035) add a space after the `!` here.

Reply:
good catch, thank you, will fix